Fix service account typos

ryanahamilton opened the pull request:
Also updates some formatting consistency.

paolaperaza commented on cloud/stable/03_deploy/06_ci-cd.md:
````diff
@@ -51,32 +51,32 @@ Read below for guidelines on how to create a service account via the CLI and via
-```sa
-$ astro workspace service-account create -d [DEPLOYMENTUUID] --label [SERVICEACCOUNTLABEL]
+```bash
+$ astro deployment service-account create -d [DEPLOYMENTUUID] --label [SERVICEACCOUNTLABEL]
````
@ryanahamilton Per [this GH issue/recent change](https://github.com/astronomer/issues/issues/1615) I think we need to replace `Deployment Uuid` with `Deployment ID`, so this should be:

```
  $ astro deployment service-account create --d <deployment-id> --label=<service-account-label> --role=<workspace-role>
```

For reference, when I run `astro deployment service-account create --help` I get:

```
 $ astro deployment service-account create --deployment-id=xxxxx --label=my_label --role=ROLE
```

A lot of our other docs have `< >` to indicate you have to "insert" something in a command (e.g. [here](https://www.astronomer.io/docs/enterprise/v0.16/manage-astronomer/registry-backend/)). We should prob standardize that - are you cool with that method or think we should go with the brackets?

paolaperaza commented on cloud/stable/03_deploy/06_ci-cd.md:
````diff
@@ -51,32 +51,32 @@ Read below for guidelines on how to create a service account via the CLI and via
 
-```sa
-$ astro workspace service-account create -w [WORKSPACEUUID] -l [SERVICEACCOUNTLABEL]
+```bash
````
Same thing here - should be `Workspace ID` I think:

```
  $ astro workspace service-account create --workspace-id=<workspace-id> --label=my_label --role=ROLE
```

paolaperaza replied:
@ryanahamilton Lol I'm the worst, but should `workspace-id` be all-caps like you have `deployment-id` above? Doesn't matter to me, but let's be consistent!

paolaperaza commented on enterprise/next/04_deploy/06_ci-cd.md:
````diff
@@ -51,32 +51,32 @@ Read below for guidelines on how to create a service account via the CLI and via
 
-```sa
-$ astro workspace service-account create -d [DEPLOYMENTUUID] --label [SERVICEACCOUNTLABEL]
+```bash
````
Apply uuid > id change here

paolaperaza commented on enterprise/next/04_deploy/06_ci-cd.md:
````diff
@@ -51,32 +51,32 @@ Read below for guidelines on how to create a service account via the CLI and via
 
-```sa
-$ astro workspace service-account create -w [WORKSPACEUUID] -l [SERVICEACCOUNTLABEL]
+```bash
````
Apply uuid > id change here

paolaperaza commented on enterprise/v0.12/04_deploy/06_ci-cd.md:
````diff
@@ -51,32 +51,32 @@ Read below for guidelines on how to create a service account via the CLI and via
 
-```sa
-$ astro workspace service-account create -d [DEPLOYMENTUUID] --label [SERVICEACCOUNTLABEL]
+```bash
````
Only Astronomer Enterprise "next" and "0.16" have the `uuid` change applied, so Enterprise versions 0.15 and lower should be kept with `uuid`

paolaperaza reviewed (commented):
Thanks for this @ryanahamilton ! Made a few comments around:

- On Astro Cloud 0.19+ and Enterprise "next" commands, `uuid` > `id` for service account creation commands in the CLI
- I don't think this is new, but we also need to add `--role=ROLE` at the end of all create commands

Otherwise looks great - thanks again.

ryanahamilton commented:
@paolaperaza I addressed all of your feedback. Here's the format I landed on:
```bash
$ astro deployment service-account create -d <deployment-id> --label <service-account-label> --role <deployment-role>
```

paolaperaza reviewed (approved):
Looks good @ryanahamilton ! Feel free to make that all-caps change to be consistent and merge whenever you're ready.

ryanahamilton commented:
Seems I missed a few! Grateful for your 🦅 👀 

After looking at some other CLI tools (Git, GitHub, Yarn, NPM), it's obvious that the `<lowercase-placeholder>` is the most common pattern.

paolaperaza commented:
Oh great, sounds good! Thanks for doing the research.
